tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -0,0 +1,252 @@
+using System;
+using System.IO;
+using System.Linq;
+using Nuke.Common;
+using Nuke.Common.IO;
+using Nuke.Common.Tools.DotNet;
+using Nuke.Common.Tooling;
+using Nuke.Common.Tools.Git;
+using static Nuke.Common.Tools.DotNet.DotNetTasks;
+
+public enum MonitoringType
```

Comment:
This name still seems too vague to me. How about `ExplorationTestType` or `ExplorationTestUseCase` ?/cc @gleocadie

Reply:
My intention was to make `MonitoringType` global enum, that represents product being tested.
If we are going to use that for exploration tests only, I think `ExplorationTestUseCase` match best.

Reply:
My vote is for `ExplorationTestUseCase` for now, until another need comes along, and then we can rename it to something else.

Reply:
Using `ExplorationTestUseCase `